cfgov/unprocessed/apps/mmt-my-money-calendar/js/stores/root-store.js

```diff
@@ -1,17 +1,15 @@
 import { observable, action, computed } from 'mobx';
-import logger from '../lib/logger';
-import UIStore from './ui-store';
-import CashFlowStore from './cash-flow-store';
+// import CashFlowStore from './cash-flow-store';
+import InputWizardStore from './input-wizard-store';
 
 export default class RootStore {
   @observable networkStatus = 'idle';
 
   constructor() {
-    this.logger = logger.addGroup('rootStore');
-    this.uiStore = new UIStore(this);
-    this.eventStore = new CashFlowStore(this);
-
-    this.logger.debug('Initialize RootStore: %O', this);
+    // this.logger = logger.addGroup('rootStore');
+    this.InputWizardStore = new InputWizardStore(this);
```

Comment:
This needs to be lowercase (`this.inputWizardStore`). It's an instance, not a class.